index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en" class=" csopmgezmow idc0_326">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <script src="https://kit.fontawesome.com/aa7a31f950.js" crossorigin="anonymous"></script>
+    <style>
+        .wrapper {
+            min-height: calc(100vh - 15px);
+            margin: 15px auto 0;
+        }
+    </style>
+    <title>LogicPart</title>
+</head>
+
+<body cz-shortcut-listen="true">
+    <div class="container-fluid wrapper">
+        <nav class="navbar navbar-light bg-light">
+            <h1>MyToDoList</h1>
+            <div class="d-flex">
+                <button id="sort-from-new" type="button" class="btn btn-primary mx-2"><i class="fas fa-sort-numeric-up-alt" aria-hidden="true"></i></button>
+                <button id="sort-from-old" type="button" class="btn btn-primary"><i class="fas fa-sort-numeric-up" aria-hidden="true"></i></button>
+
+            </div>
+            <div class="d-flex">
+                <button id="new-task" type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModal">
+                    <i class="fas fa-plus" aria-hidden="true"></i>
+                    Add task
+                </button>
+                <div class="dropdown ml-2">
+                    <button class="btn btn-primary" type="button" id="dropdownMenuSettings" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        <i class="fas fa-cog" aria-hidden="true"></i>
+                    </button>
+                    <div class="dropdown-menu p-2 dropdown-menu-right" aria-labelledby="dropdownMenuSettings">
+                        <div class="custom-control custom-switch">
+                            <input type="checkbox" class="custom-control-input" id="customSwitch1">
+                            <label class="custom-control-label" for="customSwitch1">White mode</label>

```

Comment:
do you have other custom switch ? give correct className. Avoid digits in names

Reply:
thanks!
fixed it!
```suggestion
<input type="checkbox" class="custom-control-input" id="customSwitch">
```